server/server.js

```diff
@@ -1,7 +1,7 @@
 import http from "node:http";
 
 import app from "./app";
-import { connectDb, disconnectDb } from "./db";
+import { connectDb, disconnectDb} from "./db";
```

Comment:
This file also wasn't changed, so I recommend adding the space back in to avoid merge conflicts.